### PR TITLE
 Easy CPU/GPU Arrays and Dataframes

### PR DESCRIPTION
**Pull request by quasiben**
cross posting from medium rapids blog

https://medium.com/rapids-ai/easy-cpu-gpu-arrays-and-dataframes-run-your-dask-code-where-youd-like-e349d92351d

cc @jacobtomlinson @rjzamora

### CODE REVIEW

File: _posts/2023-02-02-easy-cpu-gpu.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+>>> darr
+dask.array<randint, shape=(10, 20), dtype=int64, chunksize=(2, 5), chunktype=cupy.ndarray>
+```
+
+Now, we can leverage the `array.backend` configuration to create a CuPy-backed dask array for random data:
+
+```python
+>>> with dask.config.set({"array.backend": "cupy"}):
+...     darr = da.random.randint(0, 3, size=(10, 20), chunks=(2, 5))  # Get cupy-backed collection
+...
```
Comment (jrbourbeau) [inline _posts/2023-02-02-easy-cpu-gpu.md:54]: Noting the indention level looks normal when this markdown file is rendered on GitHub, but looks off on the NVIDIA post 

Screenshot 2023-02-01 at 2 43 59 PM

**Review by jrbourbeau — COMMENTED** [review]
Cool, thanks for writing this up @rjzamora @quasiben

**Comment by GenevieveBuckley** [comment]
Nice blogpost, and I appreciate the signal boost since I probably would have missed it on Medium 😄 

I'd like if it also included an example on how to use a gpu backend for input IO of arrays.

I've tried the equivalent of the `read_csv` and `read_parquet` examples in the blogpost (eg: using `from_zarr` and `from_tiledb` instead) but the arrays from that clearly have numpy backed chunks.

Am I doing something wrong here? 
Or is IO not yet supported for arrays? If so, the phrase *"users can optionally select the backend engine for input IO and data creation"* should probably be modified to reflect that.

**Comment by quasiben** [comment]
@GenevieveBuckley thanks for reading it over.  You're right, the from_* array support was not included.  I just submitted a WIP https://github.com/dask/dask/pull/9914 though I imagine it could be done in a significantly better way

**Comment by quasiben** [comment]
In 0ca9b0c I added a note on the missing functionality for tiledb, zarr, and from_array

File: _posts/2023-02-02-easy-cpu-gpu.md (hunk 1 above)
Comment (quasiben) [reply _posts/2023-02-02-easy-cpu-gpu.md:54]: Thanks for pointing this out @jrbourbeau .  It should be fixed now

File: _posts/2023-02-02-easy-cpu-gpu.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: Easy CPU/GPU Arrays and Dataframes
+author: Rick Zamora (NVIDIA), Ben Zaitlen (NVIDIA)
+tags: []
```
Comment (GenevieveBuckley) [inline _posts/2023-02-02-easy-cpu-gpu.md]:
```suggestion
tags: [GPU]
```

**Review by GenevieveBuckley — APPROVED** [review]
Looks good to me!

We'll add a "GPU" topic tag to the post, so people can [find it & related posts more easily](https://blog.dask.org/tags.html#GPU-ref)

**Pull request merged**